app/views/products/filter.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <div class="d-inline-flex">
+    <p> Filter By Is Active </p>
+    <% if @option=="1" %>
+      <%= link_to "False", filter_products_path(flag: "0"), class: 'btn btn-primary text-center'%>
+    <% elsif @option ="0" %>
+      <%= link_to "True", filter_products_path(flag: "1"), class: 'btn btn-primary text-center'%>
```

Comment:
Its not a good way to filter records by setting flag, like @option = 1 or 0.
Don't do it this way.